mod/metrics/set/type/metric/thumbnail.rb

```diff
@@ -18,7 +18,11 @@ def thumbnail_title
                                    view: :name),
                         class: "ellipsis")
     return content unless voo.show?(:thumbnail_link)
-    link_to_card card, content, title: card.metric_title_card.name
+    opts = { title: card.metric_title_card.name }
+    if voo.closest_live_option(:project)
+      opts[:path] = { filter: { project: voo.closest_live_option(:project) } }
+    end
```

Comment:
With the introduction of your voo API it suddenly wasn't possible anymore to pass arguments to other views. `closest_live_option` was my emergency hack to get that feature back.